test-integration/student/dashboard.coffee

```diff
@@ -0,0 +1,52 @@
+{describe, CourseSelect, Calendar, ReadingBuilder, StudentDashboard} = require '../helpers'
+{expect} = require 'chai'
+_ = require 'underscore'
+
+
+describe 'Student Dashboard', ->
+
+  @it 'Shows Performance Forecast', ->
+    @dash = new StudentDashboard(@, 'student01', 'Biology I')
```

Comment:
why `@dash =` instead of just `dash =`? Also, what's the advantage of instantiating a `new StudentDashboard` each time?